site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -483,18 +555,27 @@ The length and type of characters of the generated code used in the Change Passw
 [field]#Email Verification# [required]#Required#::
 The length and type of characters of the generated code used in the Email Verification flow.
 
+[field]#Email Verification one-time code# [required]#Required#::
+The length and type of characters of the generated code used for Email Verification one-time codes.
+
 [field]#Passwordless Login# [required]#Required#::
 The length and type of characters of the generated code used in the Passwordless Login flow.
 
 [field]#Registration Verification# [required]#Required#::
 The length and type of characters of the generated code used in the Registration Verification flow.
 
+[field]#Registration Verification one-time code# [required]#Required#::
+The length and type of characters of the generated code used for Registration Verification one-time codes.
```

Comment:
The one-time code for registration verification is only used with registration verification gating and when using a form field config instead of the clickable link.